### DISCUSSION TITLE
"edit" command doesn't work

### DISCUSSION THREAD

**Discussion by subtra3t** [nesbox/TIC-80#1529] [Q&A] [answered]
Hello. I am pretty new to TIC-80 (and fantasy consoles in general), so this might seem like a stupid question.

In [this tutorial](https://github.com/nesbox/TIC-80/wiki/A-step-by-step-introduction-to-TIC-80,-Part-1---The-Default-Cart), the creator of the tutorial says that to reveal the code editor, we need to execute the "edit" command. But when I execute it, in the browser, TIC-80 raises an error: "Unknown command:edit".

Am I doing something wrong? If so, what?

**Comment by nesbox** [comment, marked as answer]
The `edit` command is deprecated, just press ESC instead

**Reply by subtra3t** [reply]
> The `edit` command is deprecated, just press ESC instead

Thanks!

**Reply by donnaken15** [reply]
for what reason?

**Reply by WillowTheWhimsical** [reply]
> for what reason?

Probably because you could just press ESC or F1-F5 to get to the editor.

**Reply by donnaken15** [reply]
kind of interesting
almost thought removing it was exclusive to browser for some reason

**Comment by ChildishGiant** [comment]
I've updated the linked tutorial to use ESC instead.

**Comment by trufae** [comment]
uhm isnt alt+1 what needs to be pressed to go back to the editor? esc just toggles between shell and editor, but alt+1 goes directly there

**Reply by WillowTheWhimsical** [reply]
F1 also.

**Comment by andypiper** [comment]
Thanks for this, as a new user it is a bit confusing that the wiki shows `edit` as an option but it does not work.
